dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/administrative/FlowItemUtils.java

```diff
@@ -211,6 +212,12 @@ public static FlowResult processEditItem(Context context, UUID itemID, Request r
 			String lang = request.getParameter("language_"+index);
 			String remove = request.getParameter("remove_"+index);
 			
+			// Empty lang is null lang
+			if (StringUtils.isEmpty(lang))
```

Comment:
Since isEmpty() doesn't trim the input string, would isBlank() work better here? I wonder about the off chance that the user leaves a space in the field.  https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/StringUtils.html

Reply:
Well spotted.  I agree.

Reply:
This change seems like a good idea.  I do not have a repository available to test the change.

Reply:
Thanks, fixed this.